shared/js/media/media_frame.js

```diff
@@ -165,8 +164,10 @@ MediaFrame.prototype.displayImage = function displayImage(blob,
 
   // Create a blob URL for it, combine it with the media fragment for
   // downsampling, and put it in CSS background-image format.
```

Comment:
Comment at line 37 and line 166 mention media frame using  CSS background-image and should be updated